grafonnet/template.libsonnet

```
@@ -1,6 +1,25 @@
 {
   /**
-   * @name template.new
+   * Returns a new template that can be added to a dashboard.
+   * See what's a [template](https://grafana.com/docs/grafana/latest/variables/templates-and-variables/#templates).
+   *
+   * @name template.new Returns a new template.
```

```suggestion
   * @name template.new
```

Realizing the parser doesn't handle copy after the name well. It ends up being one big headline in the generated markdown. Can we drop the extra text? Seems like the description covers this anyway.

---

Yes, totally okay. I found this documentation to be redundant with the description anyway.